### PR TITLE
Redesign beautify index page

### PR DESCRIPTION
**Pull request by jethronap**

### CODE REVIEW

**Comment by jethronap** [comment]
pls check out the front page. I think it's going well. if you like it I can work a little bit more on this direction.

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+
+<div class="container">
+
+    <div class="starter-template">
+        <h1>Welcome to Jstat</h1>
+        <p>What would you like to do today?</p>
+    </div>
+    <div>
+        <ul>
+            <li><a th:href="@{/load-dataset}">Load Dataset</a></li>
```
Comment (pockerman) [inline src/main/resources/templates/index.html]: I think these should go in the header...the header manages the Navigation right? If yes then we should move them to the ```header.html```

File: src/main/resources/templates/fragments/header.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <!-- this is header -->
+    <nav class="navbar navbar-inverse">
+        <div class="container">
+            <div class="navbar-header">
+                <a class="navbar-brand" th:href="@{/}">Jstat</a>
+            </div>
+            <div id="navbar" class="collapse navbar-collapse">
+                <ul class="nav navbar-nav">
+                    <li class="active"><a th:href="@{/}">Home</a></li>
+                    <li class="active"><a th:href="@{/about}">About</a></li>
```
Comment (pockerman) [inline src/main/resources/templates/fragments/header.html]: ```active``` should be only the item selected right?

**Comment by pockerman** [comment]
I had a look at the GUI looks ok but still I think that we should move Load Dataset and Quit into the navigation bar. Furthermore, I think I am getting some sort of flashing every time I click on a link....

**Comment by jethronap** [comment]
listen, overall the main page looks ok and all your suggestions can be done. however I think that using thymeleaf as our main template engine for serving pages is a bit of an overkill.
I am working on a react project. I am about to wrap it up and start implementing the findings here.

**Comment by pockerman** [comment]
It's fine with me if you want to change the template engine...just suggest what you think we should use....

**Comment by jethronap** [comment]
I think we should use react and just have a plain index page inside spring just for rendering the react components.

**Comment by pockerman** [comment]
fair enough let's do it this way then...

**Comment by pockerman** [comment]
are we going to close this or you are planning to do the beautification here?

**Comment by jethronap** [comment]
check the new version. I think we should merge this one in master and then take it from there.
what do you think?

**Pull request merged**